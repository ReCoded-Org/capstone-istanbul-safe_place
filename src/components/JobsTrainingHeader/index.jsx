import React from "react";
import "./index.scss";

const JobsTrainingHeader = () => {
  return (
    <div className="jobsTrainingHeader">
      <h2>Job / Training Opportunities</h2>
      <h6>Browse our list of job opportunities, trainings or workshops.</h6>
    </div>
  );
};

export default JobsTrainingHeader;
