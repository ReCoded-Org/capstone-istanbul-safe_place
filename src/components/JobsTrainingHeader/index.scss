@import "../../style/variables";

.jobsTrainingHeader {
  padding-top: 1rem;

  h2 {
    color: $primaryColor;
    text-align: center;
    font-family: $mainFontFamily;
    font-weight: bold;
  }

  h6 {
    color: $darkColor;
    text-align: center;
    font-family: $mainFontFamily;
  }
}
