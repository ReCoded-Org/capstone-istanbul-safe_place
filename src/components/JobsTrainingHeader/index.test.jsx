import React from "react";
import renderer from "react-test-renderer";
import JobsTrainingHeader from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<JobsTrainingHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
