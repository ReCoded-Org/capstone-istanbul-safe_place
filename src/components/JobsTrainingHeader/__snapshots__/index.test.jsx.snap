// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="jobsTrainingHeader"
>
  <h2>
    Job / Training Opportunities
  </h2>
  <h6>
    Browse our list of job opportunities, trainings or workshops.
  </h6>
</div>
`;
